Cargo.toml

```diff
@@ -121,5 +121,5 @@ test-attestation-cert-ids = []
 # interchange = { git = "https://github.com/trussed-dev/interchange", branch = "main" }
 
 [package.metadata.docs.rs]
-features = ["ext", "virt"]
+features = ["serialized-extensions", "virt"]
```

Comment:
typo: `serde-extensions`

Reply:
Thanks, I have that on my upstream branch (and didn't intend to push that on top of your changes to this without prior discussion).